html_layout.py

```diff
@@ -418,6 +547,17 @@ def create_appLayout(
                                 id="usageCPU_input",
                                 style=dict(display='none'),
                             ),
+
+                            html.Div(
+                                [
+                                    html.Div('i', className='tooltip-icon'),
+                                    html.P(
+                                        "Between 0 and 1. Should correspond to a temporal factor usage, accessible from log files for instance.",
```

Comment:
I'm wondering if it would be worth aligning the (i) icons here (and below) on the right, same as the icons above? More generally, we could try to align all of them.